src/common-elements/_discussions.scss

```diff
@@ -1,22 +1,25 @@
+body.discussion .content-wrapper {
+  padding-top: 0;
+}
 .a--courseware-top-nav__list {
-
-  li.right {
-    margin-left: auto;
-  }
+  list-style-type: none;
 }
+.a--container {
+  .discussion-submit-post:not(:disabled),
+  .new-post-btn:not(:disabled) {
+    border-radius: $buttons-border-radius;
+    border: 0.1rem solid $brand-primary-color;
+    padding: 1rem 1.5rem;
+    font-size: 1.4rem;
+    @include transition(all 0.3s ease-in-out);
+    opacity: 1;
 
-a.new-post-btn {
-  border-radius: $buttons-border-radius;
-  border: 0.1rem solid $brand-primary-color;
-  padding: 1rem 1.5rem;
-  @include transition(all 0.3s ease-in-out);
-  opacity: 1;
-
-  span {
-    margin-right: 0.5rem;
-  }
-
-  &:hover, &:focus {
-    opacity: 0.6;
+    // Have to specify :not:(:disabled) above to win against
+    // _edx-overrides.scss
```

Comment:
c6ece21 is nicer.. it uses tag names instead of the `not(:disabled)` pseudoclass, here and for the `.sequence-nav-button`s.

Also has the added benefit of fixing the disabled nav button styles too, e.g. at the [very beginning](https://eucalyptus-theme-updates.stage.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/courseware/d8a6192ade314473a78242dfeedfbf5b/edx_introduction/) and the [very end](https://eucalyptus-theme-updates.stage.opencraft.hosting/courses/course-v1:edX+DemoX+Demo_Course/courseware/1414ffd5143b4b508f739b563ab468b7/workflow/) of the course.